converter/src/main/java/gov/cms/qpp/conversion/decode/QrdaDecoder.java

```diff
@@ -0,0 +1,98 @@
+package gov.cms.qpp.conversion.decode;
+
+import com.google.common.base.Strings;
+import org.jdom2.Attribute;
+import org.jdom2.Element;
+import org.jdom2.Namespace;
+import org.jdom2.filter.Filter;
+import org.jdom2.filter.Filters;
+import org.jdom2.xpath.XPathExpression;
+import org.jdom2.xpath.XPathFactory;
+
+import gov.cms.qpp.conversion.Context;
+import gov.cms.qpp.conversion.correlation.PathCorrelator;
+import gov.cms.qpp.conversion.model.Decoder;
+import gov.cms.qpp.conversion.model.Node;
+import gov.cms.qpp.conversion.model.validation.SupplementalData;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+import static gov.cms.qpp.conversion.decode.SupplementalDataEthnicityDecoder.SUPPLEMENTAL_DATA_CODE;
+import static gov.cms.qpp.conversion.decode.SupplementalDataEthnicityDecoder.SUPPLEMENTAL_DATA_KEY;
+import static gov.cms.qpp.conversion.decode.SupplementalDataPayerDecoder.SUPPLEMENTAL_DATA_PAYER_CODE;
+
+public abstract class QrdaDecoder {
+
+	protected final Context context;
+	private Namespace defaultNs;
+	protected Namespace xpathNs;
+
+	public QrdaDecoder(final Context context) {
+		this.context = context;
+	}
+
+	protected abstract DecodeResult decode(Element element, Node thisNode);
+
+	/**
+	 * Sets xml namespace
```

Comment:
could you add some more notes here about what setting the xml namespace will affect?